### DISCUSSION TITLE
How to truncate text inside a Flex?

### DISCUSSION THREAD

**Discussion by nik72619c** [chakra-ui/chakra-ui#5594] [Q&A]
Hi,
Using `Text` component with `isTruncated` prop inside `Flex` component does not seem to work. 

Tried reproducing the issue with this code snippet in the code editor in chakra - ui's website:-
```
<Flex color='white'>
  <Flex></Flex>
  <Flex><Text isTruncated>sharmasharmasharmasharmasharmasharma sharmasharmasharmasharma</Text>
</Flex>
</Flex>
<img width="1440" alt="Screenshot 2022-02-16 at 6 13 44 PM" src="https://user-images.githubusercontent.com/30630904/154267851-1398bd1c-9664-456b-ab53-092ba502fd9e.png">

**Comment by nik72619c** [comment]
We need to apply a min-width on the flex item but seems like Flex component in chakra-ui does not have this built-in 🤔 

<img width="1440" alt="Screenshot 2022-02-16 at 6 13 44 PM" src="https://user-images.githubusercontent.com/30630904/154268121-5f73f98f-1906-4624-8113-e27c6fa09214.png">

**Comment by TylerAPfledderer** [comment]
Hey @nik72619c!

Have you tried specifying a `width` or `max-width` to the `Flex` component?
